zebra-consensus/src/transaction.rs

```diff
@@ -497,12 +479,11 @@ where
                 // the resulting future to our collection of async
                 // checks that (at a minimum) must pass for the
                 // transaction to verify.
-                let output_rsp = output_verifier
-                    .ready_and()
-                    .await?
-                    .call(primitives::groth16::ItemWrapper::from(output).into());
-
-                async_checks.push(output_rsp.boxed());
+                async_checks.push(
+                    primitives::groth16::OUTPUT_VERIFIER
+                        .clone()
+                        .oneshot(primitives::groth16::ItemWrapper::from(output).into()),
+                );
```

Comment:
Same here, is doing all this cloning  what we what?

Reply:
Yeah, it's only a few `Arc`s per clone, but it's not optimal. This could be improved later using `call_all().unordered()`, but will likely need some thought and experimentation 😅

Reply:
OK cool, we can revisit later